sdk/python/v1beta1/kubeflow/katib/constants/constants.py

```diff
@@ -13,15 +13,28 @@
 # limitations under the License.
 
 import os
+from typing import Final
+
+# How long to wait in seconds for requests to the ApiServer
+APISERVER_TIMEOUT: Final[int] = 120
 
 # Global CRD version
-KATIB_VERSION = os.environ.get('EXPERIMENT_VERSION', 'v1beta1')
+KATIB_VERSION: Final[str] = os.environ.get("EXPERIMENT_VERSION", "v1beta1")
 
 # Katib K8S constants
-KUBEFLOW_GROUP = 'kubeflow.org'
-EXPERIMENT_PLURAL = 'experiments'
-SUGGESTION_PLURAL = 'suggestions'
-TRIAL_PLURAL = 'trials'
+KUBEFLOW_GROUP: Final[str] = "kubeflow.org"
+EXPERIMENT_KIND: Final[str] = "Experiment"
+EXPERIMENT_PLURAL: Final[str] = "experiments"
+SUGGESTION_PLURAL: Final[str] = "suggestions"
+TRIAL_PLURAL: Final[str] = "trials"
 
-# How long to wait in seconds for requests to the ApiServer
-APISERVER_TIMEOUT = 120
+
+DEFAULT_PRIMARY_CONTAINER_NAME: Final[str] = "training-container"
+
+# Supported base images for the Katib Trials.
+# TODO (andreyvelich): Implement list_base_images function to get each image description.
+BASE_IMAGE_TENSORFLOW: Final[str] = "docker.io/tensorflow/tensorflow:2.9.1"
```

Comment:
Should we add the base image for tensorflow with GPU support same as the tfjob client?